sources/__init.sh

```diff
@@ -13,13 +28,25 @@ function __init {
         echo "If you continue, stored options will be overridden!"
         echo
         read -r -p "Do you want to proceed? [y/n] " c
-        if ! [[ ${c} =~ ^(yes|y|Y) ]] || [ -z ${c} ]; then
-            exit 1
+        if [[ ${c} =~ ^(yes|y|Y) ]] || [ -z ${c} ]; then
+           __init_deploy_options
         fi
```

Comment:
If the user enters N or just hits enter, what would happen if this logic is removed? Will execution stop?